R/plot_recurrent_fusions.R

```diff
@@ -58,18 +85,23 @@ plot_recurrent_fusions <- function(standardFusioncalls,
     palette_1 <- palette[which(names(palette) %in% rec_fusions[, groupby])]
   }
 
+  if (base_size != 12) {
+    base_size <- base_size
+  }
+
   rec_fusions$FusionName <- factor(rec_fusions$FusionName, levels = unique(rec_fusions$FusionName), ordered = TRUE)
   rec_fusions_plot <- ggplot(rec_fusions) +
     geom_col(aes(x = FusionName, y = count, fill = !!as.name(groupby)), alpha = 0.75) +
     guides(alpha = FALSE) +
-    ylab("Number of patients") +
+    ylab(paste0("Number of ", countID)) +
     xlab(NULL) +
     guides(color = FALSE, alpha = FALSE) +
-    scale_y_continuous(breaks = seq(0, 200, by = 20)) +
+    scale_y_continuous(breaks = seq(0, max(rec_fusions$count), by = 20)) +
```

Comment:
I might also suggest not to do the `by` because we won't know what other data looks like
```suggestion
    scale_y_continuous(limits=(0, max(rec_fusions$count))) +
```